src/Locale/Locale.elm

```diff
@@ -0,0 +1,38 @@
+module Locale.Locale exposing (..)
+
+import Locale.English as English
+import Locale.Portuguese as Portuguese
+import Locale.Types exposing (Language(..), LocaleKey)
+
+
+fromCode : String -> Language
+fromCode code =
+    case code of
+        "pt-BR" ->
```

Comment:
`pt-PT` is also portuguese :P

Reply:
I believe it can also come as just `pt` as well, maybe we should split at the `-` and only use the first part

Reply:
yeah, it is also Portuguese, but I believe there are a lot of different words and the way of phrases are build is different too, so it is maybe better to have a separated translation for them 😄  What do you think about?

Reply:
well, it is not *that* much different, I believe it is better for a portuguese user to see it in pt-BR than in english

Reply:
same will happen with en (en-US, en-UK, en-AU, ...) and es (es-CL, es-AR, es-MX, ...), it is better to try use a more generic form of the languages than to add all variations

Reply:
Awesome, I'll handle with just the first part of string 😄